var/ramble/repos/builtin/workflow_managers/slurm/workflow_manager.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2022-2024 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+from ramble.wmkit import *
+from ramble.expander import ExpanderError
+
+
+_ensure_job_id_snippet = r"""
+job_id=$(< {experiment_run_dir}/.slurm_job)
+if [ -z "${job_id:-}"]; then
+    echo "No valid job_id found" 1>&2
+    exit 1
+fi
+"""
+
+
+class Slurm(WorkflowManagerBase):
+    """Slurm workflow manager"""
+
+    name = "slurm"
+
+    maintainers("linsword13")
+
+    tags("workflow", "slurm")
+
+    render_content(
+        name="batch_submit",
```

Comment:
How does this work when we define `batch_submit` in the variables? Does this override the definition of the `batch_submit` variable?

Reply:
Yeah it does override the `batch_submit` variable. I do that so that `ramble on` works without needing to specify an `executor`. Do you see this being undesirable in some cases?

Reply:
Nope, I was more concerned that the variable would override the template.